CMakeLists.txt

```diff
@@ -59,15 +59,15 @@ include(cmake/snappy.cmake)
 include(cmake/lz4.cmake)
 include(cmake/rocksdb.cmake)
 include(cmake/libevent.cmake)
-include(cmake/lua.cmake)
+include(cmake/luajit.cmake)
```

Comment:
I think we can provide a build option to switch from luajit and lua for progressive substitution.